package.json

```diff
@@ -69,5 +75,16 @@
       "npm run eslint-check",
       "git add"
     ]
+  },
+  "jest": {
```

Comment:
Not an error, but it's better to put jest configuration in separate file

Reply:
Ok, thanks